src/init/features/dataconnect/sdk.ts

```diff
@@ -141,6 +141,7 @@ async function askQuestions(setup: Setup, config: Config): Promise<SDKInfo> {
     const javascriptSdk: JavascriptSDK = {
       outputDir,
       package: pkg,
+      packageJsonDir: newConnectorYaml.generate.javascriptSdk?.packageJsonDir,
```

Comment:
I remember there being a prompt for this at some point elsewhere in the code actually